### PR TITLE
add applyToRound to allo.ts and wire it into builder for v1

### PR DESCRIPTION
**Pull request by codenamejason**
Flows to test:

-  [x] Apply to round still works on v1
-  [x] View the application

Fixes: #2787 

The next PR, will update builder to apply to a round created on v2

### CODE REVIEW

File: packages/common/src/allo/abis/allo-v2/Strategy.ts [hunk 1]
```diff
@@ -0,0 +1,356 @@
+export default [
```
Comment (thelostone-mc) [inline packages/common/src/allo/abis/allo-v2/Strategy.ts:1]: Is this needed ? Since we are relying on the sdk -> can this be removed ?

Comment (codenamejason) [reply packages/common/src/allo/abis/allo-v2/Strategy.ts:1]: yes!! good catch!

Comment (codenamejason) [reply packages/common/src/allo/abis/allo-v2/Strategy.ts:1]: Just noticed that we use the abi in the test. We could just hardcode that event signature.

**Review by michellema1208 — APPROVED** [review]
LGTM - both core flows worked for me!

**Review by thelostone-mc — APPROVED** [review]

**Review by 0xKurt — APPROVED** [review]

**Pull request merged**